docs/application/native/guides/compliations/watchface-complication.md

```diff
@@ -193,113 +187,111 @@ In This case, watchface supports complications only in allowed complication list
-```
- 
-1. When the editor is ready to edit, it notifies the edit ready. To receive the notification from the editor, watchface_editable_edit_ready_cb() should
-be added.
+To use the functions and data types for the watchface_editable API, include the `<watchface-editable.h>` header file in your application:
 
-```cpp
-void _watchface_editable_edit_ready_cb(watchface_editable_container_h handle,
-		const char *editor_appid,
-		void *user_data)
-{
-}
+    ```cpp
+    #include <watchface-editable.h>
+    ```
 
-{
-	watchface_editable_add_edit_ready_cb(_watchface_editable_edit_ready_cb, NULL);
-}
-```
+1. When the editor is ready to edit, it notifies that it is ready. To receive the notification from the editor, `watchface_editable_edit_ready_cb()` must be added:
+
+    ```cpp
+    void _watchface_editable_edit_ready_cb(watchface_editable_container_h handle,
+    		const char *editor_appid,
+    		void *user_data)
+    {
+    }
+
+    {
+    	watchface_editable_add_edit_ready_cb(_watchface_editable_edit_ready_cb, NULL);
+    }
+    ```
  
-2. Watchface can make the complication_candidates_list_h to identify the editable features to request to editor.
-Watchface can request to edit the features by using watchface_editable_request_edit().
+2. Watchface can create the `complication_candidates_list_h` to identify the editable features to request to editor.
+Watchface can request to edit the features by using `watchface_editable_request_edit()`.
 
-3. The watchface editor can draw highlight to give the information what is editing.
-To inform the position of editable features, watchface_editable_highlight_h should be used.
-The highlight handle can be created by watchface_editable_hilight_create() function with watchface shape type.
+3. The watchface editor can draw highlight to give the information about editing.
+To inform the position of editable features, `watchface_editable_highlight_h` must be used.
+The highlight handle can be created by `watchface_editable_highlight_create()` with watchface shape type.
 
-There are following shape types.
+There are following shape types:
 
-| Shape types | Description |
-|-------------|-------------|
-| WATCHFACE_EDITABLE_SHAPE_TYPE_CIRCLE | circle |
-| WATCHFACE_EDITABLE_SHAPE_TYPE_RECT | rectangle |
+    | Shape types | Description |
+    |-------------|-------------|
+    | `WATCHFACE_EDITABLE_SHAPE_TYPE_CIRCLE` | circle |
+    | `WATCHFACE_EDITABLE_SHAPE_TYPE_RECT` | rectangle |
 
 
-The position of editable feature can be set by using watchface_editable_highlight_set_geometry().
-The parameters of this function mean the position(x,y) and size(width, height).
+The position of editable feature can be set by using `watchface_editable_highlight_set_geometry()`.
+The parameters of this function means the position(x,y) and size(width, height).
 
 
+    > **Note**
+    >
+    > `watchface_editable_highlight_h` must be released after used by using `watchface_editable_highlight_destroy()`.
 

```

Comment:
Please donnot add space after > for note style

Reply:
please add note in the following format:

![image](https://user-images.githubusercontent.com/35525967/47892488-df9a1580-de4f-11e8-9806-cb2d723f334d.png)

Reply:
Please change as : 
`watchface_editable_highlight_h` must be released after using `watchface_editable_highlight_destroy()`.

Please check the one that is highlighted, please add it: 
![image](https://user-images.githubusercontent.com/35525967/47895443-360f5000-de60-11e8-9347-97d67870c048.png)